File: Round1/Bit N Bytes Rating/code.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner as =new Scanner(System.in);
        int R = as.nextInt();
        if(R<1200)
            System.out.println("BNB-Beginner");
        else if(R<2800)
            System.out.println("BNB-Intermediate");
        else
            System.out.println("BNB-All");
    }
}

File: Round1/Bit N Bytes Rating/code.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int R;
    cin>>R;
    if(R<1200){
        cout<<"BNB-Beginner"<<endl;
    }else if(R<2800){
        cout<<"BNB-Intermediate"<<endl;
    }else{
        cout<<"BNB-All"<<endl;
    }
    return 0;
}